index.html

```diff
@@ -73,253 +79,420 @@ <h1 class="page-background" id="principal-title" class="color-blue fonts">I’m
+          work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+      </div>
 
 
-          <ul id="contactInGrid">
-            <li>
-              <div class="input-div">
-                <input type="text" name="name" maxlength="30" required="true" placeholder="Your Name" />
-              </div>
-            </li>
-            <li>
-              <div class="input-div">
-                <input id="mail" type="email" name="email" required="true" placeholder="Your email Address"
-                  maxlength="500" />
-              </div>
-            </li>
-          </ul>
+      <div class="contact position contact2 position2">
+        <h3>LET'S CONNECT</h3>
+        <ul class="social width half-width">
+          <li class="about-li">
+            <a href="https://twitter.com/lerfast" class="a3">
+              <img class="img-about" src="./vectors/twitter Vector.svg" alt="Github logo">
+            </a>
+          </li>
+          <li class="about-li">
+            <a href="https://www.linkedin.com/in/luis-emilio-rojas-rojas-6ba90a8a/" class="a3">
+              <img class="img-about" src="./vectors/Linkedin icon (1).png" alt="Linkedin logo">
+            </a>
+          </li>
+          <li class="about-li">
+            <a href="https://medium.com/" class="a3">
+              <img class="img-about" src="./vectors/middle.png" alt="hmedium-icon">
+            </a>
+          </li>
+          <li class="about-li">
+            <a href="https://github.com/lerfast" class="a3">
+              <img class="img-about" src="./vectors/github logo.png" alt="Twitter logo">
+            </a>
+          </li>
+          <li class="about-li">
+            <a href="https://www.angellist.com/" class="a3">
+              <img class="img-about" src="./vectors/angellist-icon.png" alt="Twitter logo">
+            </a>
           </li>
+        </ul>
+        <a href="" class="see quarter-width">Get my
+          resume</a>
+      </div>
+    </div>
 
-          <div id="maxText" class="input-div">
-            <textarea name="message" required="true" id="message" maxlength="500"
-              placeholder="Write your message here..." rows="10"></textarea>
-          </div>
+
+    <ul class="skills">
+      <li class="skill">
+        <div class="name">
+          <p>Languages</p>
+
+
+        </div>
+        <ul class="languages-list">
+          <li class="about-li">
+            <img class="img2-about" src="./vectors/Ellipse 1.png" alt="JavaScript">
+            JavaScript
           </li>
+          <li>
+            <img class="img2-about" src="./vectors/HTML Ellipse 1.png" alt="HTML">
+            HTML
+          </li>
+          <li>
+            <img class="img2-about" src="./vectors/css Ellipse 1.png" alt="CSS">
+            CSS
+          </li>
+        </ul>
+      </li>
+      <li class="skill">
+        <div class="name">
+          <p>Frameworks</p>
```

Comment:
- [ ] **[OPTIONAL]** Please make sure to update the about section so the list of skills will match the figma design.
**Your project**
![image](https://user-images.githubusercontent.com/74411135/229290867-ef7628a4-a448-4208-b6fb-ad386e92f553.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229290882-62bf003c-9fb2-4e74-be1e-8f04c60400e5.png)

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪